src/spin.jl

```diff
@@ -192,3 +191,8 @@ function Base.:*(si::Spin,sj::Spin)
         return QMul(1, [sj,si])
     end
 end
+
+# axis(s::Spin) = s.axis
```

Comment:
Is it intentional that this is commented out?

Reply:
I just forgot to delete it. Thanks!